src/server/tablet_snap.rs

```diff
@@ -378,6 +384,9 @@ impl<R: RaftExtension> Runnable for TabletRunner<R> {
         match task {
             Task::Recv { stream, sink } => {
                 let task_num = self.recving_count.load(Ordering::SeqCst);
+                STORE_SNAPSHOT_TRAFFIC_GAUGE_VEC
+                    .with_label_values(&["receiving"])
+                    .set(task_num as i64);
```

Comment:
The metrics will never go back to 0 again.

Reply:
good catch

Reply:
I have move it to the store heartbeat and impl snapshot stats in next pr.